R/import_functions.R

```diff
@@ -3523,7 +3534,8 @@ determinand.link.sum <- function(data, info, keep, drop, ...) {
       out$censoring <- unique(x$censoring) 
     else 
       out$censoring <- "<"
-
+      out$censoring <- if(all(x$censoring %in% "<")) "<" else ""
```

Comment:
This is a legacy bit of code that shouldn't be there.  My fault.  It is the original code that was updated by the lines above when censoring started taking the values D and Q as well as < and "".  I obviously didn't get rid of it!.  Can you remove this line, make another commit, and then I'll approve the pull request.

Reply:
@RobFryer 
I can certainly remove it. Since you do not specify which line you mean, I just want to double check that it is line 3557 you are referring to in this case?

Reply:
Yes, it's line 3537.  I totally agree, it's not clear.  When I added the comment to the code, I only selected line 3537 and hoped that selection would be clear, but it definitely isn't!
Speak later